SketchUp/Geom/BoundingBox.rb

```diff
@@ -253,10 +249,10 @@ def initialize
   #   boundingboxes actually overlap - then call this to get the resulting
   #   boundingbox.
   #
-  # @param boundingbox
+  # @param boundingbox [Geom::BoundingBox]
```

Comment:
The Documentation Guide mentions it vice versa (that's why I followed it consistently) and gives a logical explanation to separate the name from the sentence without hyphen.
`@param name [types] Full sentence`
If I understand right, the source from where you generate the docs has it the other way, so the question is whether following the old or the new format creates more consistency or inconsistency?